functions/src/CVE/index.ts

```diff
@@ -60,6 +60,9 @@ export const getCVE = functions.https.onRequest((request, response) => {
     }
     getCvesWithAndroidVersion(String(androidVersion), response);
   }
+  else{
+    response.status(400).send("Error: A query parameter is required.");
```

Comment:
You may want to be more specific, i.e. "no valid query parameters specified. "valid parameters are ..."